plugins/docker-machine/_docker-machine

```diff
@@ -169,7 +169,7 @@ __get_create_argument() {
 __docker-machine_subcommand() {
     local -a opts_help
     opts_help=("(- :)--help[Print usage]")
-    local -a opts_only_host opts_driver opts_storage_driver opts_stragery
+    local -a opts_only_host opts_driver opts_storage_driver opts_state
```

Comment:
This is different. I can't find any sign of `opts_stragery` it felt like `opts_strategy` but that wasn't around. `opts_state` otoh wasn't listed as a local.

This could be very wrong.

Reply:
It doesn't matter, it was supposed to be used when the completion was first introduced but it passed the review. It should be deleted but since `docker-machine` is deprecated and it's a local variable that doesn't do anything this can be ignored.